internal/cli/api/api.go

```diff
@@ -99,6 +99,14 @@ func createAPICommandGroupToCobraCommand(group shared_api.Group) *cobra.Command
 func convertAPIToCobraCommand(command shared_api.Command) (*cobra.Command, error) {
 	// command properties
 	commandName := strcase.ToLowerCamel(command.OperationID)
+	commandAliases := command.Aliases
+
+	// Prefer to use shortOperationID if present
+	if command.ShortOperationID != "" {
+		commandName = strcase.ToLowerCamel(command.ShortOperationID)
+		commandAliases = append(commandAliases, strcase.ToLowerCamel(command.OperationID))
```

Comment:
Small nit, but why not re-use the variable you created in the previous line, makes it easier to follow:
```suggestion
		commandAliases = append(commandAliases, commandName)
```